lib/flipper/views/layout.erb

```diff
@@ -60,5 +60,13 @@
         <%== yield %>
       </div>
     </div>
+    <script>
+    const baseUrl = '<%= Settings.hostname %>' === '127.0.0.1:3000' ? 'localhost:3000': '<%= Settings.hostname %>';
```

Comment:
Does there need to be a space before the ternary colon? (idk what the JS style is 😅)
```suggestion
    const baseUrl = '<%= Settings.hostname %>' === '127.0.0.1:3000' ? 'localhost:3000' : '<%= Settings.hostname %>';
```

Reply:
I don't think it matters honestly. I can rewrite it this way too: `const baseUrl = '<%= Settings.hostname %>'.replace('127.0.0.1:3000', 'localhost:3000');`, it just depends on what the BE would prefer 🤷

Reply:
Cool, if you don't think it matters then I don't either